docs/01-app/04-api-reference/04-functions/use-link-status.mdx

```diff
@@ -89,17 +89,33 @@ In this example, navigating between categories updates the query string (e.g. ?c
 
 You can use the `useLinkStatus` hook to render a lightweight loading indicator next to the active link and provide immediate feedback to the user while the data is being fetched.
 
-```tsx filename="app/page.tsx" switcher
+```tsx filename="app/loading-indicator.tsx" switcher
```

Comment:
Just thinking this might clarify this is different than our special `loading.tsx` component

```suggestion
```tsx filename="app/components/loading-indicator.tsx" switcher
```